cfgov/jinja2/v1/_includes/molecules/featured-menu-content.html

```diff
@@ -0,0 +1,43 @@
+{# ==========================================================================
+
+   featured_menu_content
+
+   ==========================================================================
+
+   Description:
+
+   Create a featured content molecule. See
+   [GHE]/flapjack/Modules-V1/wiki/Featured-Menu-Content
+
+   value: An object with the following options for value.
+
+   value.body:  Body text.
```

Comment:
I was aligning the same level e.g. (`value.body` and `value.link`; `value.link.url` and `value.link.text`), which we do sometimes elsewhere, but there's space for them all to align here, will fix. Also, actually I didn't align body and link :stuck_out_tongue: